examples/image_expansion.py

```diff
@@ -0,0 +1,104 @@
+"""
```

Comment:
This script is quite dense. Please break it up into logical pieces with blank lines.

Reply:
done.